waiter/src/waiter/simulator.clj

```diff
@@ -53,6 +53,7 @@
   Stats:
     total-queue-time       The total amount of ticks that clients have been sitting in the queue.
     total-idle-server-time The total amount of ticks that servers have been sitting idle.
+    utilization            The percentage of servers that are in use during the current tick.
```

Comment:
servers or slots?

Reply:
The sim doesn't support concurrency (it pre-dates concurrency support) and refers to servers elsewhere.